tests/network/services.go

```diff
@@ -58,21 +58,6 @@ const (
 var _ = SIGDescribe("Services", func() {
 	var virtClient kubecli.KubevirtClient
 
-	cleanupVMI := func(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) {
```

Comment:
Please mention in the commit message that the VMI cleanup was dropped.
As already mentioned - I would drop the jo and service clean as well. The general test cleanup will take care of it anyway.